### PR TITLE
Add DSLogon notification banner for current outage

### PR DESCRIPTION
**Pull request by edpaget**
Using text from: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9390

Screenshot: 
screenshot from 2018-04-26 12-38-54

### CODE REVIEW

**Comment by edpaget** [comment]
Updated Screenshot: 
screenshot from 2018-04-26 12-55-08

File: src/platform/site-wide/login/components/SignInModal.jsx [hunk 1]
```diff
@@ -44,6 +45,18 @@ class SignInModal extends React.Component {
               <h2>One site. A lifetime of benefits and services at your fingertips.</h2>
             </div>
           </div>
+          <div className="downtime-notification row">
+            <div className="columns usa-width-one-half medium-6">
+              <div className="form-warning-banner">
+                <AlertBox
+                  headline={<h3>DS Logon isn't working quite right</h3>}
```
Comment (jbalboni) [inline src/platform/site-wide/login/components/SignInModal.jsx]: I think the headline is already wrapped in an `h3`.

File: src/platform/site-wide/login/components/SignInModal.jsx [hunk 2]
```diff
@@ -44,6 +45,18 @@ class SignInModal extends React.Component {
...
+                  content={<p>If you're having trouble signing in to Vets.gov using your DS Logon ID and password, please try again later. Or, you can try signing in with your MyHealth_e_Vet ID and password or through ID.me.</p>}
```
Comment (jbalboni) [inline src/platform/site-wide/login/components/SignInModal.jsx]: The `p` element I don't think should be necessary, but doesn't hurt anything either way.

**Review by jbalboni — APPROVED** [review]
LGTM, but I'm not a login expert.

**Review by ncksllvn — APPROVED** [review]

**Pull request merged**